content/docs/integrations/pagerduty/index.md

```diff
@@ -10,43 +10,100 @@ integrationType: OSS plugin
 
 ## Introduction
 
+The PagerDuty Backstage plugin allows Roadie Backstage users to:
+
+ 1. See who is on call for a particular catalog Component.
+ 2. See if there are active incidents for a given catalog Component.
+ 3. Create new incidents from Backstage.
+
+![PagerDuty plugin in Roadie Backstage](./pagerduty-plugin.png)
+
```

Comment:
Added this information so the reader understands the value they will get from completing these steps.

Reply:
👍